packages/@lwc/engine-server/src/__tests__/fixtures/render-correct-tags/expected.html

```diff
@@ -0,0 +1,42 @@
+<x-elements class="lwc-4q9u0sqsfc0-host" data-lwc-host-scope-token="lwc-4q9u0sqsfc0-host">
+  <template shadowrootmode="open">
+    <style class="lwc-4q9u0sqsfc0" type="text/css">
+      :host {background: blue;}
+    </style>
+    <a class="lwc-4q9u0sqsfc0" href="https://www.salesforce.com/">
+    </a>
+    <label class="lwc-4q9u0sqsfc0" for="textInput">
+      Some input
+    </label>
+    <input class="lwc-4q9u0sqsfc0" id="textInput" type="text">
+    <svg class="lwc-4q9u0sqsfc0">
+      <pattern class="lwc-4q9u0sqsfc0">
+        <image class="lwc-4q9u0sqsfc0" xlink:href="https://www.salesforce.com/"/>
+      </pattern>
+      <image class="lwc-4q9u0sqsfc0" xlink:title="title"/>
+    </svg>
+    <math class="lwc-4q9u0sqsfc0">
+      <mi class="lwc-4q9u0sqsfc0">
+        π
+      </mi>
+      <mo class="lwc-4q9u0sqsfc0">
+        ⁢
+      </mo>
```

Comment:
It'd be great to have an example of a self-closing tag in MathML to make sure we do it correctly.

Also `&InvisibleTimes;` is very confusing here since it renders invisibly. Tricked me into thinking that this should be self-closing when it isn't.

Reply:
Wait – this should be self-closing, right? Since it's actually just a comment?

Reply:
> Wait – this should be self-closing, right? Since it's actually just a comment?

I'm not sure... it is actually appearing as an invisible character aka ZWJ (yellow box) in chrome and renders that? It passes in both engine-server and ssr-compiler (not to say that means it is correct). It is a confusing example though.

> It'd be great to have an example of a self-closing tag in MathML to make sure we do it correctly.

Sorry I didn't add that. Unfortunately we have another `engine-server` bug here... 
- `<plus/>` and `<plus></plus>` is rendered as `<plus></plus>` by `engine-server`
- `<plus/>` and `<plus></plus>` is rendered as `<plus/>` by `ssr-compiler`

Shall I open another issue?

Reply:
I missed that there is literally an invisible character in there, outside of the comment. Yeah I would switch to something that's visible.

> Shall I open another issue?

Yes, I would open a bug for this. All non-HTML childless components should render as self-closing (with the `/>`).

Reply:
Ok sounds good, opened another issue on engine-server, referenced it in the example and removed the invisible character example